assets/js/modules/analytics/components/common/ExistingGTMPropertyNotice.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Analytics Existing GTM Property Notice component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { sprintf, __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../datastore/constants';
+import { STORE_NAME as MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
+const { useSelect } = Data;
+
+export default function ExistingGTMPropertyNotice() {
+	const existingTag = useSelect( ( select ) => select( STORE_NAME ).getExistingTag() );
+	const gtmAnalyticsPropertyID = useSelect( ( select ) => select( MODULES_TAGMANAGER ).getSingleAnalyticsPropertyID() );
+	const gtmAnalyticsPropertyIDPermission = useSelect( ( select ) => select( STORE_NAME ).hasTagPermission( gtmAnalyticsPropertyID ) );
+
+	// Don't display this notice if:
+	if (
+		existingTag || // There is an existing tag.
```

Comment:
I think we should remove this from here too and handle that part of visibility in `SettingsForm` and `SetupForm` instead, because this component shouldn't need to worry about an existing tag.

Reply:
Updated.